titiler/pgstac/mosaic.py

```diff
@@ -364,7 +364,9 @@ def _reader(
         if "allowed_exceptions" not in kwargs:
             kwargs.update({"allowed_exceptions": (PointOutsideBounds,)})
 
-        return list(multi_values(mosaic_assets, _reader, lon, lat, **kwargs).items())
```

Comment:
I had to go away from `multi_values` because the STAC item metadata was getting set to the dictionary keys and I was getting an `unhashable` error.